package.json

```diff
@@ -23,7 +23,6 @@
   "workspaces": [
     "packages/renderers/*",
     "packages/*",
-    "examples/*",
```

Comment:
Should this change be included in a "Docs sync"? I feel like this should be it's own thing, if small. Otherwise, LGTM! Since this is just copying over the current `astro-docs` content

Reply:
oh yea, not sure how that got in here. Thanks for catching, will revert!